### ISSUE TITLE
[Bug]: Subtitles not automatically displayed

### ISSUE THREAD

**Issue by Bananeaa** [ThaUnknown/miru#430]
### Preflight checklist

- [X] I have searched the [issue tracker](https://github.com/ThaUnknown/miru/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [frequently asked questions](https://github.com/ThaUnknown/miru/blob/master/docs/faq.md) for a solution that fixes this problem, without success.
- [X] I have checked that I'm using the [latest stable](https://github.com/ThaUnknown/miru/releases/latest) version of the app.

### What app version are you using?

v4.5.10

### What operating system version are you using?

MacOS

### Expected Behavior

When starting an episode, subtitles are displayed automatically without adjusting the subtitle option settings in episode

### Actual Behavior

Subtitles are automatically selected on "End - Sign/Songs" instead of "Jap - English", despite options in app being restored to default. How to prioritise the second option permanently without changing the option in the episode settings for each episode ?

**Comment by ThaUnknown** [comment]
Miru automatically selects whichever track the torrent labels as "Default", this means the person who originally made the release manually specified that the preferred track for playback is "Sign/Songs", questionable, but it's not my fault, you can cycle tru subtitle tracks with a keybind, by default C

There's also default language selection for audio tracks, the release you're watching labeled the english subtitles as japanese, and since miru prefers english it chose the non-jap ones

TLDR bad release.
